### PR TITLE
chore(tooling): improve stack traces

### PR DESCRIPTION
**Pull request by ojongerius**
Addresses #57

### CODE REVIEW

File: _webpack/include.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// Add source map support for proper stack traces
+import 'source-map-support/register';
+
+// Catch all unhandled exceptions and print their stack trace.
+// Required if the handler function is async, as promises
+// can swallow error messages.
+process.on('unhandledRejection', e => {
```
Comment (Bouncey) [inline _webpack/include.js:7]: I would print the entire `stringified` error object, seeing as it was an unexpected error, the `error.message` would help out in debugging too.

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: _webpack/include.js (hunk 1 above)
Comment (ojongerius) [reply _webpack/include.js:7]: AFAICT e.stack will include both stack and message:

```js
> let terribleError = new Error('oh noes!');
> terribleError.toString()
'Error: oh noes!'
> terribleError.message
'oh noes!'
> console.log(terribleError.stack)
Error: oh noes!
    at repl:1:21
    at ContextifyScript.Script.runInThisContext (vm.js:50:33)
    at REPLServer.defaultEval (repl.js:240:29)
    at bound (domain.js:301:14)
    at REPLServer.runBound [as eval] (domain.js:314:12)
    at REPLServer.onLine (repl.js:468:10)
    at emitOne (events.js:121:20)
    at REPLServer.emit (events.js:211:7)
    at REPLServer.Interface._onLine (readline.js:282:10)
    at REPLServer.Interface._line (readline.js:631:8)
```

**Review by raisedadead — APPROVED** [review]
LGTM. Just two things:

1. Prefer bringing in changes from a fork. 
2. Always use a rebase for updating any pull requests instead of a merge from staging

```
git pull --rebase upstream staging
git push origin
```

Reason: 
We avoid bad / accidental / extra commits in the history.

**Comment by raisedadead** [comment]
P.S: I am not fully sure but, I think we have an issue on our main production site where some dependencies are conflicting even if a PR did not update the packaging at all.

Hence another recommendation to squash and merge.

**Comment by ojongerius** [comment]
Ok,  I'll behave and will raise a PR from my fork 😄

**Comment by raisedadead** [comment]
Oh no.. you did not have to close this one though... Don't want to make you work on this again.

**Comment by raisedadead** [comment]
Please re-open if this is not trivial.

**Comment by ojongerius** [comment]
That's alright, it was faster then fixing merge conflicts 😁

**Pull request closed without merging**